sdk/eventhub/tests.yml

```diff
@@ -11,6 +11,18 @@ jobs:
   - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: eventhub
+      Linux_Python35:
```

Comment:
Please change formatting: 

```
...
    parameters:
      ServiceDirectory: eventhub
      Matrix: 
        Linux_Python35:
          OSName: 'Linux'
          OSVmImage: 'ubuntu-16.04'
          PythonVersion: '3.5'
        MacOs_Python37:
          OSName: 'MacOS'
          OSVmImage: 'macOS-10.14'
          PythonVersion: '3.7'
        Windows_Python27:
          OSName: 'Windows'
          OSVmImage: 'windows-2019'
          PythonVersion: '2.7'
      EnvVars:	     ...
```

The matrix entries should be under a `Matrix` block

Reply:
Thanks Daniel. It's working now after adding Matrix.